app/src/main/java/de/xikolo/models/Item.java

```diff
@@ -112,6 +115,26 @@ public int getIconRes() {
         return icon;
     }
 
+    public String formatTimeEffort() {
+        int hours = timeEffort / 3600;
+        int minutes = (int) Math.ceil((timeEffort % 3600) / 60.0);
+
+        if (hours > 0) {
+            if (minutes > 0) {
+                return App.getInstance().getString(R.string.item_duration_hour_minute, hours, minutes);
+            } else {
+                return App.getInstance().getString(R.string.item_duration_hour, hours);
+            }
+        }
+        if (hours == 0) {
+            if (minutes > 0) {
```

Comment:
This could be a simple `else if` statement without the need to check the hours again.

Reply:
done, also fixed the string formatting of the other cases